f2/server.js

```diff
@@ -230,7 +230,7 @@ app
         logger.error('ERROR', err)
         throw err
       }
-      submitFeedback(fields.json)
+      submitFeedback(sanitize(fields.json))
```

Comment:
Why sanitize here?

Reply:
Not sure why it's there(I probably had a mistake with github). I update the branch and it's gone. Thanks anyway!